src/services/hmwk-service.js

```diff
@@ -17,27 +17,32 @@ class HmwkService {
         }
       }`;
 
-    const variables = { studentsBoardId };
+    const response = await monday.api(query);
 
-    const response = await monday.api(query, { variables });
-    const studentInfo = response.data.boards.items;
+    // Since we are only querying 1 single students Board), index 0 at boards array
+    const studentInfo = response.data.boards[0].items;
     return studentInfo; //array of json
     //student name: student_info[i].name
     //student email: student_info[i].column_values.text
   }
 
   static async createNewHmwk(hmwkCompletionTrackingBoardId, hmwkName) {
```

Comment:
I haven't tested this function yet but here I am just fixing a few lines that I know are going to break...